### PR TITLE
Fixed Typos in README.md issue #228

### PR DESCRIPTION
**Pull request by anuragsachdeva28**
Fixed issue #228 [https://github.com/Nexmo/nexmo-node/issues/228]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ nexmo.applications.get(appId, callback);
 
 For more information see https://developer.nexmo.com/api/application#retrieve-an-application
 
-### Get Apps by filter
+### Get Apps by the filter
```
Comment (AlexLakatos) [inline README.md:323]: I'm not a native english speaker, but `by the filter` doesn't sound right to me. Can we make this "Get Apps with a filter`?

Comment (anuragsachdeva28) [reply README.md:323]: According to common grammar rules, before the word filter, there must be an article. It can either be 'the' or 'a'.

Comment (AlexLakatos) [reply README.md:323]: make it `a` then please

Comment (anuragsachdeva28) [reply README.md:323]: Updated the proposed change. Link to updated pull request is: https://github.com/Nexmo/nexmo-node/pull/231

**Pull request merged**